integration-tests/models/test_lora_mistral.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+import requests
+
+
+@pytest.fixture(scope="module")
+def lora_mistral_handle(launcher):
+    with launcher(
+        "mistralai/Mistral-7B-v0.1",
+        lora_adapters=[
+            "predibase/dbpedia",
+            "predibase/customer_support",
+        ],
+        cuda_graphs=[0],
```

Comment:
Why ?

Reply:
currently theres a bug with lora and cuda graphs but wanted to make sure there were basic tests for expected responses. I'll push a update for cuda graphs soon along with some of the other refactoring changes. Happy to keep this open until it's resolved

Reply:
What is the bug ?

Does load with many different loader breaks things ?

Reply:
currently the adapter is ignored because now adapter_data is passed to model.forward and is not inlcuded in warmup. I need to update the graph warmup to include adapter data and also replace the values before calling replay